vhdbuilder/packer/init-variables.sh

```diff
@@ -113,6 +113,14 @@ if [ -z "${SUBNET_NAME}" ]; then
 	SUBNET_NAME="packer"
 fi
 
+USE_SHALLOW_REPLICATION="false"
+if [[ "${OS_TYPE,,}" == "linux" ]] && [[ "${ENVIRONMENT,,}" == "test" ]]; then
```

Comment:
to make sure we can deal with the inability to replicate to other regions, I recommend having a new boolean pipeline parameter in `.vsts-vhd-builder.yaml`/`.vsts-vhd-builder-release.yaml` which would determine whether or not to use shallow replication, which we can default to true

if we ever need to run abe2e's against a particular build in a _different_ region than where the build takes place, we can simply disable shallow replication with this new parameter

Reply:
Sounds good. Will DM.